google-cloud-firestore/src/main/java/com/google/cloud/firestore/BulkWriterOptions.java

```diff
@@ -20,15 +20,31 @@
 
 /** Options used to disable request throttling in BulkWriter. */
 final class BulkWriterOptions {
+  static final double DEFAULT_UNSET_VALUE = 1.1;
+  private final boolean throttling;
+  private double initialOpsPerSecond = DEFAULT_UNSET_VALUE;
+  private double maxOpsPerSecond = DEFAULT_UNSET_VALUE;
 
-  private final boolean enableThrottling;
+  BulkWriterOptions(boolean enableThrottling) {
```

Comment:
Does this need to be package-private now?

Reply:
N/A w/ AutoValue.